assets/airports/lrop.json

```diff
@@ -0,0 +1,1150 @@
+{
+    "radio": {
+        "twr": "Otopeni Tower",
+        "app": "Bucharest Approach",
+        "dep": "Bucharest Radar"
+    },
+    "icao": "LROP",
+    "iata": "OTP",
+    "magnetic_north": 5,
+    "ctr_radius": 180,
+    "ctr_ceiling": 17500,
+    "initial_alt": 6000,
+    "position": ["N44d34m16.00", "E26d05m06.00", "314ft"],
+    "rr_radius_nm": 5,
+    "rr_center": ["N44d34m16.00", "E26d05m06.00"],
+    "has_terrain": true, 
+    "wind": {
+        "angle": 90,
+        "speed": 10
+    },
+    "airspace": [{
+        "floor": 20,
+        "ceiling": 175,
+        "airspace_class": "A",
+        "poly": [
+            ["N45d04m34.000", "E25d11m30.000"],
+            ["N45d04m34.000", "E26d27m15.000"],
+            ["N44d53m23.000", "E26d58m01.000"],
+            ["N44d20m00.000", "E27d00m00.000"],
+            ["N44d00m00.000", "E25d40m00.000"],
+            ["N44d04m47.000", "E25d15m11.000"],
+            ["N44d37m37.000", "E25d08m58.000"]
+        ]
+    },
+    {
+        "floor": 45,
+        "ceiling": 175,
+        "airspace_class": "A",
+        "poly": [
+            ["N45d04m34.000", "E25d11m30.000"],
+            ["N45d04m34.000", "E26d27m15.000"],
+            ["N45d01m22.000", "E26d36m08.000"],
+            ["N44d48m00.000", "E25d58m00.000"],
+            ["N44d55m05.000", "E25d10m36.000"]
+        ]
+    }],
+    "restricted": [{
+        "name": "LRR3 - Bucharest",
+        "height": "10500ft",
+        "coordinates": [
+            ["N44d30m25.000", "E26d14m00.000"],
+            ["N44d27m20.000", "E25d58m30.000"],
+            ["N44d21m00.000", "E26d04m00.000"],
+            ["N44d21m00.000", "E26d14m00.000"]
+       ]
+    },
+    {
+        "name": "LRD100 - Clinceni",
```

Comment:
This is a danger area, aircraft are discouraged from flying in it, but it is still allowed.

Reply:
that is a parachute jumping area, when active atc actually vector the aircraft around the area.
you say I should totally remove it?

Reply:
You know more about the local procedures than I do, so it's up to you really. Normally I wouldn't include danger areas, but if you think it should be left in, then that's fine too!